themes/catppuccin.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "./schema.json",
+  "author": {
+    "name": "enno",
+    "twitter": "@YOUR TWITTER HANDLE",
```

Comment:
Can you add your twitter username or remove this property, please?
```suggestion
    "twitter": "@YOUR TWITTER HANDLE",
```

Reply:
Fixed!